src/NFont.cs

```diff
@@ -0,0 +1,387 @@
+﻿using System;
+using SDL2;
+
+namespace SDL2
+{
```

Comment:
The library in question doesn't compile to a DLL, unfortunately, so this is a straight translation, not a wrapper.  It's still a useful SDL satellite library.